oneandone/src/main/java/org/apache/jclouds/oneandone/rest/domain/LoadBalancer.java

```diff
@@ -51,7 +51,7 @@
    public abstract String healthCheckPath();
 
    @Nullable
-   public abstract Object healthCheckPathParser();
+   public abstract String healthCheckPathParser();
```

Comment:
If this is a random json object, then you'd better map it to the [JsonBall](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/domain/JsonBall.java) type.